test/bdd/features/w3c_chapi_http.feature

```diff
@@ -22,7 +22,7 @@ Feature: CHAPI Workflow - API only
     Then  "<verifier>" verifies the DIDAuth response from "<holderProfile>"
     Then  "<verifier>" verifies the "<flow>" presented by "<holderProfile>"
     Examples:
-      | flow                        | issuerProfile         | holderProfile     | verifier        | credential                       |
-      | Education Degree            | university            | alice             | acme            | university_degree.json           |
-      | Permanent Resident Card     | government            | bob               | immigrationDept | permanent_resident_card.json     |
-      | Certified Mill Test Report  | qualityinc            | stark_industries  | customsDept     | certified_mill_test_report.json  |
+      | flow                        | issuerProfile         | holderProfile     | verifier        | credential                       | issuerKeyType | issuerSignType        | holderKeyType | holderSignType          |
+      | Education Degree            | university            | alice             | acme            | university_degree.json           | Ed25519       | Ed25519Signature2018  | Ed25519       | Ed25519Signature2018    |
+      | Permanent Resident Card     | government            | bob               | immigrationDept | permanent_resident_card.json     | Ed25519       | Ed25519Signature2018  | P256          | JsonWebSignature2020    |
```

Comment:
can we do this one twice?

Ed25519       | Ed25519Signature2018  | P256          | JsonWebSignature2020
Ed25519       | Ed25519Signature2018  | Ed25519    | Ed25519Signature2018

Reply:
[Added](https://github.com/trustbloc/edge-service/compare/ca82fdd7348bab09a6b5922d945432058693b4a3..d0275929986e8d2ab551105485d9e643daccfac0#diff-464ee9eb2c810dd01b535ac7455e84f0R27-R28).